UPGRADE.md

```diff
@@ -0,0 +1,81 @@
+# Upgrade Guide
+
+_All `MAJOR` version bumps will have upgrade notes posted here._
+
+[2022-04-27] 0.2.x to 1.x.xrc
```

Comment:
This needs to be updated to the date when the actual RC version is released for librarian to be able to find it.